specifyweb/backend/workbench/upload/tests/test_batch_edit_table.py

```
@@ -402,6 +404,262 @@ def _make_data():
+                    "id": record["species"].pk,
+                    "ordernumber": None,
+                    "version": record["species"].version
+                }
+            }
+        return actual_record
+
+    def _rank_to_formatted(self, rank_name: str, include_id: bool = False):
+        tree_id = self.taxontreedef.pk
+        tree_name = self.taxontreedef.name
+        final = [tree_name, rank_name]
+        if include_id == True:
```

Ruff (0.16.3)

[error] Avoid equality comparisons to `True`; use `include_id:` for truth checks

Replace with `include_id`

(E712)